lib/CXGN/Stock/Search.pm

```diff
@@ -721,9 +765,6 @@ sub _refresh_materialized_stockprop {
         my $h = $schema->storage->dbh()->prepare($q);
 
         my $stockprop_refresh_q = "
-        DROP EXTENSION IF EXISTS tablefunc CASCADE;
-        CREATE EXTENSION tablefunc;
-
```

Comment:
@MFlores2021 @nmenda thoughts on this?

Reply:
@timparsons this might be required for refreshing the stockprop materialized view. Can you check if the refresh function works without the drop/create extension?